odoo/models.py

```diff
@@ -1367,6 +1367,10 @@ def load_views(self, views, options=None):
         :param options['action_id']: id of the action to get the filters
         :return: dictionary with fields_views, fields and optionally filters
         """
+        # populate the context with essential information for debugging problematic views
+        # all the way up the stack
+        self = self.with_context(loaded_view=[self._name, views, options])
```

Comment:
That sucks.  Looks like a lazy way to fix a problem :wink:

Reply:
The alternative would be passing down these 3 parameters down to `fields_view_get`, `post_process_and_fields` and then to `post_process` (which will be called recursively), given that and the fact that the task is supposed to be backportable, I thought this was the best approach